src/coverlet.core/Helpers/InstrumentationHelper.cs

```diff
@@ -40,10 +78,11 @@ public static bool HasPdb(string module)
             }
         }
 
-        public static void BackupOriginalModule(string module, string identifier)
+        public static string BackupOriginalModule(string module, string identifier)
```

Comment:
Oh! One more question here. Why did you update this to return a string as I don't see it used anywhere

Reply:
@jherby2k changed that to simplify testing: https://github.com/tonerdo/coverlet/pull/253/files#diff-009f00622e86f8b17a0f74fc89bd66a6R31

Reply:
I'm open to revert the change if you wish so.

Reply:
Please do, thanks!

Reply:
Done. I also added the `overwrite: true` parameter as in local testing when debugging I hit that once. I hope that's fine.

Reply:
That's fine